selfdrive/car/tesla/carcontroller.py

```diff
@@ -321,14 +322,20 @@ def update(self, enabled, CS, frame, actuators, \
     if (frame % 5 == 0):
       if (changing_lanes and not self.prev_changing_lanes): #we have a transition from blinkers off to blinkers on, save the frame
         self.ldw_numb_frame_start = frame
-        print("LDW Transition detected, frame (%d)", frame)
-
+        if (CS.v_ego > self.ldw_enable_speed_ms):
```

Comment:
It's not idiomatic to use parentheses around `if` conditions in Python unless you're trying to disambiguate operator precedence